graph/is_graph_bipartite.cpp

```diff
@@ -37,127 +37,124 @@
  * @namespace graph
  * @brief Graph algorithms
  */
-namespace graph{
-	/**
-	 * @namespace is_graph_bipartite
-	 * @brief Functions for checking whether a graph is bipartite or not
-	 */
-	namespace is_graph_bipartite{
-		/**
-		 * @brief Class for representing graph as an adjacency list.
-		 */
-		class Graph {
- 			private: 
- 				int n;						/// size of the graph
-
-	    		std::vector<std::vector <int> > adj;	/// adj stores the graph as an adjacency list
-
-		    	std::vector<int> side;			///stores the side of the vertex
+namespace graph {
+  /**
+   * @namespace is_graph_bipartite
+   * @brief Functions for checking whether a graph is bipartite or not
+   */
+  namespace is_graph_bipartite {
+    /**
+     * @brief Class for representing graph as an adjacency list.
+     */
+    class Graph {
+      private:
+        int n;            /// size of the graph
 
-				static const int nax = 5e5 + 1;
+        std::vector< std::vector<int> > adj;  /// adj stores the graph as an adjacency list
 
+        std::vector<int> side;      ///stores the side of the vertex
 
-	 		public:
-	    		/**
-    			 * @brief Constructor that initializes the graph on creation
-    			 */
-	    		explicit Graph(int size = nax){
-    				n = size;
-    				adj.resize(n);
-	    			side.resize(n,-1);
-    			}
+        static const int nax = 5e5 + 1;
```

Comment:
```suggestion
        static const int max = 5e5 + 1;  ///< ....
```
can also use `INT_MAX`